src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -1,74 +1,38 @@
 ---
 linkTitle: Creating a Grafana organization
 title: Creating a Grafana organization
-description: Guide explaining how to manage Grafana organizations in the Observability Platform.
+description: Step-by-step guide to create and configure Grafana organizations for multi-tenant observability.
 menu:
   principal:
     identifier: tutorials-observability-multitenancy-create-grafana-organization
     parent: tutorials-observability-multitenancy
 weight: 40
 last_review_date: 2025-06-11
 user_questions:
-  - How to create a grafana organization?
-  - How to access multi-tenant observability data?
+  - How to create a Grafana organization?
+  - How to configure RBAC for Grafana organizations?
+  - How to assign tenants to organizations?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-atlas
 ---
 
-When you first log in to [your installation's `Grafana`]({{< relref "/tutorials/observability/data-exploration/accessing-grafana" >}}), you see the preselected `Grafana` organization called `_Shared Org_`. This shared organization contains a curated set of managed dashboards available to everyone with `Grafana` access.
+This guide walks you through creating and configuring Grafana organizations to implement [multi-tenancy]({{< relref "/tutorials/observability/multi-tenancy" >}}) in your observability platform.
 
-If multiple teams access the observability platform, you should take advantage of multi-tenancy to isolate data and dashboards. The observability platform lets you create new organizations in self-service for this use case.
+## Prerequisites
 
-## Understanding tenants and organizations
+Before creating your organization, make sure you have:
 
-Before creating your organization, it's important to understand the relationship between **tenants**, **organizations**, and **RBAC groups**:
+- Access to the management cluster, where you can create custom resources
+- Identified the [tenant names]({{< relref "/tutorials/observability/multi-tenancy#tenant-naming-best-practices" >}}) you want to use
+- Configured identity provider groups for RBAC
```

Comment:
Is there a doc for that?

Reply:
not really, this will most likely be for shield I think